src/ActionModal/components/UndelegationModal.vue

```diff
@@ -21,7 +21,7 @@
     <TmFormGroup class="action-modal-form-group">
       <div class="form-message notice">
         <span v-if="!isRedelegation">
-          Unstaking takes {{ undelegationPeriod }} to complete and cannot be
+          Unstaking takes {{ currentNetwork.lockUpPeriod }} to complete and cannot be
```

Comment:
the api is not released yet

Reply:
it is on staging

Reply:
it is not released yet to production. if we release the PR it will fail